tests/samplers_tests/test_cmaes.py

```diff
@@ -61,6 +61,33 @@ def test_init_cmaes_opts(use_separable_cma: bool, cma_class_str: str) -> None:
         assert actual_kwargs["population_size"] is None
 
 
+@patch("optuna.samplers._cmaes.get_warm_start_mgd")
```

Comment:
Could you suppress the `ExperimentalWarning`?
```suggestion
@pytest.mark.filterwarnings("ignore::optuna.exceptions.ExperimentalWarning")
@patch("optuna.samplers._cmaes.get_warm_start_mgd")
```